src/activate_linux.c

```diff
@@ -95,27 +102,14 @@ int main(int argc, char *argv[]) {
     switch (argc) {
         // if there are no arguments (1 is for program name)
         case (1):
-            #ifdef __APPLE__
-                title = "Activate macOS";
```

Comment:
Both "macOS" and "MacOS" are used in the code, but "macOS" is the proper capitalization so I'm guessing it should be kept? Up for debate though :smile:

Reply:
we should probably use macOS